crates/aptos-faucet/core/src/checkers/redis_ratelimit.rs

```diff
@@ -305,6 +305,7 @@ impl CheckerTrait for RedisRatelimitChecker {
 
     /// All we have to do here is decrement the counter if the request was a failure due
     /// to something wrong on our end.
+    #[allow(dependency_on_unit_never_type_fallback)]
```

Comment:
Allowing this seems fine to me for now, the proper never type (`!`) isn't even stable yet, we'll revisit it later when it is.